numpy/core/tests/test_regression.py

```diff
@@ -47,8 +47,10 @@ def test_pickle_transposed(self):
 
     def test_typeNA(self):
         # Ticket #31
```

Comment:
Change this to `gh-515` while you're here, maybe

Reply:
fixed